CITATION

```diff
@@ -6,10 +6,46 @@ Please cite this tutorial as:
 
 and cite ESMValTool as:
 
+- Righi, M., Andela, B., Eyring, V., Lauer, A., Predoi, V., Schlund, M.,
+  Vegas-Regidor, J., Bock, L., Brötz, B., de Mora, L., Diblen, F., Dreyer, L.,
+  Drost, N., Earnshaw, P., Hassler, B., Koldunov, N., Little, B., Loosveldt Tomas,
+  S., and Zimmermann, K.:
+  Earth System Model Evaluation Tool (ESMValTool) v2.0 – technical overview,
+  Geosci. Model Dev., 13, 1179–1199, https://doi.org/10.5194/gmd-13-1179-2020,2020.
+
+Additional resources:
+
 - Lauer, A., Eyring, V., Bellprat, O., Bock, L., Gier, B. K., Hunter, A.,
   Lorenz, R., Pérez-Zanón, N., Righi, M., Schlund, M., Senftleben, D., Weigel,
   K., and Zechlau, S.: Earth System Model Evaluation Tool (ESMValTool) v2.0 –
   diagnostics for emergent constraints and future projections from Earth
   system models in CMIP, Geosci. Model Dev. Discuss.,
   [https://doi.org/10.5194/gmd-2020-60](https://doi.org/10.5194/gmd-2020-60),
   in review, 2020.
+
+- Eyring, V., Bock, L., Lauer, A., Righi, M., Schlund, M., Andela, B., Arnone, E.,
+  Bellprat, O., Brötz, B., Caron, L.-P., Carvalhais, N., Cionni, I., Cortesi, N., Crezee, B.,
+  Davin, E., Davini, P., Debeire, K., de Mora, L., Deser, C., Docquier, D., Earnshaw, P.,
+  Ehbrecht, C., Gier, B. K., Gonzalez-Reviriego, N., Goodman, P., Hagemann, S.,
+  Hardiman, S., Hassler, B., Hunter, A., Kadow, C., Kindermann, S., Koirala, S.,
+  Koldunov, N. V., Lejeune, Q., Lembo, V., Lovato, T., Lucarini, V., Massonnet, F.,
+  Müller, B., Pandde, A., Pérez-Zanón, N., Phillips, A., Predoi, V., Russell, J., Sellar, A.,
+  Serva, F., Stacke, T., Swaminathan, R., Torralba, V., Vegas-Regidor, J., von Hardenberg, J.,
+  Weigel, K., and Zimmermann, K.:
+  ESMValTool v2.0 – Extended set of large-scale diagnostics for quasi-operational
+  and comprehensive evaluation of Earth system models in CMIP,
+  Geosci. Model Dev. Discuss.,
+  [https://doi.org/10.5194/gmd-2019-291](https://doi.org/10.5194/gmd-2019-291),
+   in review, 2019.
```

Comment:
```suggestion
  [ https://doi.org/10.5194/gmd-13-3383-2020]( https://doi.org/10.5194/gmd-13-3383-2020),
   2020.
```
this paper has been published now, maybe move it above the other one?

Reply:
You can find an up to date list of papers here: https://www.esmvaltool.org/references.html

Reply:
@bouweandela thank you. I replaced the additional resources with the link to up to date list of papers. It avoids duplication.